content/400-reference/200-api-reference/050-prisma-client-reference.mdx

```diff
@@ -5296,6 +5296,13 @@ See: [Raw database access (`$runCommandRaw()`)](/concepts/components/prisma-clie
 
 See: [Transactions](/concepts/components/prisma-client/transactions).
 
+<!-- prettier-ignore -->
+### <inlinecode>$metrics()</inlinecode>
```

Comment:
This is a readonly property, not a function

```suggestion
### <inlinecode>$metrics</inlinecode>
```

Reply:
We should probably then add sub sections for the actual functions? The reference should be complete.

Reply:
We can choose to either list both methods, with their relevant info, or as Jan suggests, have `metrics` as a title, and then have sub-sections for the two methods. There are pros and cons to each.

Reply:
I've added details of the two methods. I don't think there's a huge amount to add, but they're there for the sake of completeness.